interchip-Autopilot/Core/Inc/Interchip.hpp

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
You meant to get rid of all these files, right ?

Reply:
did not realize I commited those, whoops